src/components/navigation/hooks/useItemsWithCustomDropdowns.ts

```diff
@@ -0,0 +1,33 @@
+import {NavigationItemModel} from '@gravity-ui/page-constructor';
+
+type CustomNavigationItemModel = NavigationItemModel & {type: string};
+
+const useItemsWithCustomDropdowns = (
+    leftItems: NavigationItemModel[],
+    rightItems?: NavigationItemModel[],
+) => {
+    return [addMobileDropdownsTo(leftItems), rightItems ? addMobileDropdownsTo(rightItems) : []];
+};
+
+function addMobileDropdownsTo(items: CustomNavigationItemModel[]) {
+    const itemsWithCustomDropdowns: CustomNavigationItemModel[] = [];
+
+    items.forEach((item) => {
+        itemsWithCustomDropdowns.push(item);
+
+        if (item.type !== 'dropdown') {
+            return;
+        }
+
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        const newMobileItem: any = {};
```

Comment:
If possible, it is better not to use any

Reply:
This is really necessary. And I added a comment to make it clear why 'any' is used there

Reply:
UPD: you were right, you can do without it, now I have found a way, and I have already fixed it